lib/bundler/cli/outdated.rb

```diff
@@ -133,7 +129,7 @@ def run
 
             unless options[:parseable]
               if groups
-                Bundler.ui.info "===== Group #{groups} ====="
+                Bundler.ui.info "===== Group#{groups.split(",").size > 1 ? "s" : ""} #{groups} ====="
```

Comment:
This should be put into a variable before, to help keep the code clean.

Reply:
I will put in a method to no duplicate code.